api/src/auth.rs

```diff
@@ -156,28 +157,22 @@ async fn generate_access_token(
       .next(),
   };
 
-  let user_id = match db.get_user_by_github_id(gh_user.id).await? {
-    Some(db_user) => db_user.id,
-    None => {
-      let name = gh_user.name.unwrap_or(gh_user.login);
-      let user = NewUser {
-        name: name.as_str(),
-        email: gh_email.as_deref(),
-        avatar_url: gh_user.avatar_url.as_str(),
-        github_id: Some(gh_user.id),
-        is_blocked: false,
-        is_staff: false,
-      };
-
-      db.insert_user(user).await?.id
-    }
+  let new_user = NewUser {
+    name: name.as_str(),
+    email: gh_email.as_deref(),
+    avatar_url: gh_user.avatar_url.as_str(),
+    github_id: Some(gh_user.id),
+    is_blocked: false,
+    is_staff: false,
```

Comment:
I might be reading this wrong, wouldn't that mean we'd always set these two values to `false` regardless of whether the user is staff or is blocked upon login?

Reply:
oh good point!